pages/user/[username]/index.tsx

```diff
@@ -111,20 +112,10 @@ export async function handleUserSSR({ params }: GetServerSidePropsContext<{ user
     user = (await req.json()) as DbUser;
   }
 
-  async function fetchSocialCardURL() {
-    const socialCardUrl = `${String(process.env.NEXT_PUBLIC_OPENGRAPH_URL ?? "")}/users/${username}`;
-    const ogReq = await fetch(`${socialCardUrl}/metadata`); //status returned: 204 or 304 or 404
-    if (ogReq.status !== 204) {
-      fetch(socialCardUrl, {
-        method: "HEAD",
-      }); // trigger the generation of the social card
-    }
-
-    ogImage = ogReq.headers.get("x-amz-meta-location");
-  }
-
   // Runs the data fetching in parallel. Decreases the loading time by 50%.
-  await Promise.allSettled([fetchUserData(), fetchSocialCardURL()]);
+  const [, ogData] = await Promise.allSettled([fetchUserData(), fetchSocialCard(`users/${username}`)]);
```

Comment:
Maybe naive question, but the empty destructured item on the left a normal practice for variables or is this an oversight?
```
  const [, ogData] =...

```

Reply:
It's normal practice in the case that you only want the second item in the returned array. For clarity we could just flip the two functions calls.

```ts
const [ogData] = Promise.allSettled([fetchSocialCard(), fetchUserData()]);
```